docs/feature-flags/sdks/index.md

```diff
@@ -12,6 +12,20 @@ Eppo's client SDK integrates with client-side applications that run on a user de
 
 ![client-sdk-diagram](/img/connecting-data/client-sdk-diagram.png)
 
+### Removing duplicate assignment logs
+
+Each invocation of the `get*Assignment` methods triggers the logging callback function. 
+If you have connected it to transmit events to your Data Warehouse this 
+will store duplicate data and increase associated costs unnecessarily.
+
+Eppo's SDK deduplicates assignment events using an internal cache
+that takes into account changes in subject variations and other factors to ensure
+one and only one canonical event is transmitted.
```

Comment:
This is not quite true right: we don't ensure only 1 event is transmitted, but rather drastically reduce the number of duplicate events

Reply:
Ok I'll write it in more words but the goal is only one event is sent only when it needs to be; this might mean "duplicate" but for example if a variation changes 5 times then we need to know that.

Reply:
Yup yup agreed. I'm just saying that there might not be a guarantee that there is only 1 unique message sent (e.g. what if the second request goes to a different server and hence that one does not have the cache entry)

Reply:
not to be pedantic but this is the client section so we do expect to almost completely eliminate duplicates here 😄

Reply:
Hah okay move my comment to the server side SDK section 😂